libs/features/texts/src/acore-text/select-acore-string.integration.spec.ts

```diff
@@ -0,0 +1,140 @@
+import { TestBed, waitForAsync } from '@angular/core/testing';
+import { Router } from '@angular/router';
+import { MysqlQueryService } from '@keira/shared/db-layer';
+import { SelectPageObject, TranslateTestingModule } from '@keira/shared/test-utils';
+import { ACORE_STRING_ENTRY } from '@keira/shared/acore-world-model';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { ToastrModule } from 'ngx-toastr';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { of } from 'rxjs';
+import { SelectAcoreStringComponent } from './select-acore-string.component';
+import { AcoreStringHandlerService } from './acore-string-handler.service';
+import { SelectAcoreStringService } from './select-acore-string.service';
+
+describe(`${SelectAcoreStringComponent.name} integration tests`, () => {
+  class Page extends SelectPageObject<SelectAcoreStringComponent> {
+    override ID_FIELD = ACORE_STRING_ENTRY;
+  }
+
+  const value = 1200;
+  const expectedRoute = 'texts/acore-string';
+
+  beforeEach(waitForAsync(() => {
+    TestBed.configureTestingModule({
+      imports: [BrowserAnimationsModule, ToastrModule.forRoot(), ModalModule.forRoot(), SelectAcoreStringComponent, TranslateTestingModule],
+      providers: [AcoreStringHandlerService],
+    }).compileComponents();
+  }));
+
+  function setup() {
+    const navigateSpy = spyOn(TestBed.inject(Router), 'navigate');
+    const queryService = TestBed.inject(MysqlQueryService);
+    const querySpy = spyOn(queryService, 'query').and.returnValue(of([{ max: 1 }]));
+
+    const selectService = TestBed.inject(SelectAcoreStringService);
+
+    const fixture = TestBed.createComponent(SelectAcoreStringComponent);
+    const page = new Page(fixture);
+    const component = fixture.componentInstance;
+    fixture.autoDetectChanges(true);
+    fixture.detectChanges();
+
+    return { component, fixture, selectService, broadcast: page, queryService, querySpy, navigateSpy };
```

Comment:
why is the page being renamed to `broadcast` here?

Reply:
it shouldnt. my bad.